src/test/regress/citus_tests/run_test.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import pathlib
+from glob import glob
+import argparse
+import shutil
+import random
+import re
+import common
+import config
+
+args = argparse.ArgumentParser()
+args.add_argument("-t", "--test", required=True, help="Relative path for test file (must have a .sql or .spec extension)", type=pathlib.Path)
```

Comment:
Does this argument really need a flag? Can we allow `./run_test.py some_test` instead of `./run_test.py -t some_test`